.travis.yml

```diff
@@ -6,6 +6,7 @@ otp_release:
 sudo: true
 before_script:
   - sudo pip install crossbar
+script: "mix test && mix spell.example.pubsub"
```

Comment:
you don't need to do `mix test` travis run it for you, you need to hook `mix spell.example.pubsub` in the `after_script` callback